fili-core/src/main/java/com/yahoo/bard/webservice/util/Utils.java

```diff
@@ -220,4 +222,28 @@ public static void emitField(JsonNode node, String fieldName, ObjectMapper mappe
     public static <T extends Comparable> T getMinValue(T one, T two) {
         return one.compareTo(two) < 1 ? one : two;
     }
+
+    /**
+     *
+     *
+     * @param pair  Immutable Pair instance
+     * @param right  The right value, may be null
+     *
+     * @return New instance of Immutable Pair
+     */
```

Comment:
This `JavaDoc` should either be deleted, or given a method. It's lonely all by itself. 😿